data_summaries/SeaBench.json

```diff
@@ -0,0 +1,58 @@
+{
+    "seabench": {
+        "Unique Dataset Identifier": "seabench",
+        "Dataset Name": "SeaBench",
+        "Paper Title": "SeaLLMs - Large Language Models for Southeast Asia",
+        "Dataset URL": "https://huggingface.co/datasets/SeaLLMs/Sea-bench/raw/main/question.jsonl",
+        "GitHub URL": "https://github.com/DAMO-NLP-SG/SeaLLMs",
+        "Hugging Face URL": "https://huggingface.co/datasets/SeaLLMs/Sea-bench",
+        "ArXiv URL": "https://arxiv.org/abs/2312.00738",
+        "Semantic Scholar Corpus ID": "265551745",
+        "Collection": "SeaBench",
+        "Languages": [
+            "Tagalog",
+            "Khmer",
+            "Vietnamese",
+            "Thai",
+            "Lao",
+            "Burmese",
+            "Indonesian",
+            "Standard Malay",
+            "English"
+        ],
+        "Task Categories": [
+            "Summarization",
+            "Translation",
+            "Question Answering"
+        ],
+        "Text Sources": [
+            "commoncrawl.org",
+            "local websites",
+            "annotation from native speaker annotators"
+        ],
+        "Model Generated": [
+            "SeaLLM"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Creators": [
+            "DAMO Academy Alibaba Group"
+        ],
+        "Licenses": [
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://huggingface.co/datasets/SeaLLMs/Sea-bench"
```

Comment:
Do they ever say more about the sources they taken this data from? We should clarify that if the sources are not retrievable from the paper/documentation that we are putting Apache 2 as their own license, since they appear to be dataset creators.

Reply:
The only specific sources they mentioned in the [technical report](https://arxiv.org/pdf/2312.00738.pdf) are Common Crawl, CC News (unknown license), and Wikipedia. Unfortunately it's a bit sparse on the details re: data sources. I think putting it as Apache 2 should work.